### PR TITLE
Adds Norinori

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
New abnormality, sprited by Kojka Kill and Earthen
Will kill you if you try to pet it, unless you meet the temperance requirement, where it's qliphoth counter will dip instead.
Will kill you if you attempt attachment work.
On a breach, can deploy itself as a turret of sorts, raising its attack range to 3 and becoming more resistant to most attacks, and immune to ranged attacks.

## Why It's Good For The Game
Is it?

## Changelog
:cl:
add: Added Norinori, a new abnormality.
/:cl:

### CODE REVIEW

**Comment by TravisAngeI** [comment]
i get the feeling it's gonna look very pettable based off the gimmick it has lol

File: code/modules/mob/living/simple_animal/abnormality/he/norinori.dm [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+	see_in_dark = 10
+	stat_attack = HARD_CRIT
+	move_to_delay = 7
+	threat_level = HE_LEVEL
+	can_breach = TRUE
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 40,
+						ABNORMALITY_WORK_INSIGHT = 60,
+						ABNORMALITY_WORK_ATTACHMENT = -80,
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/he/norinori.dm:31]: He hates you, but not as much as General Bee hates you

File: code/modules/mob/living/simple_animal/abnormality/he/norinori.dm [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	user.Stun(3 SECONDS)
+	step_towards(user, src)
+	SLEEP_CHECK_DEATH(0.5 SECONDS)
+	step_towards(user, src)
+	SLEEP_CHECK_DEATH(0.5 SECONDS)
+	user.attack_animal(src)
+	SLEEP_CHECK_DEATH(0.2 SECONDS)
+	user.attack_animal(src)
+	SLEEP_CHECK_DEATH(0.5 SECONDS)
+	to_chat(user, "<span class='userdanger'>[src] mutilates you!</span>")
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/he/norinori.dm:133]:
```suggestion
	user.visible_message("<span class='warning'>[src] mutilates [user]!</span>", "<span class='userdanger'>[src] mutilates you!</span>")
```
Let people know what's happenin

File: code/modules/mob/living/simple_animal/abnormality/he/norinori.dm [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	user.apply_damage(3000, RED_DAMAGE, null, user.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+	playsound(user, 'sound/abnormalities/helper/attack.ogg', 100, FALSE, 4)
+	attack_sound = initial(attack_sound)
+	user.gib() //update it with cool effects
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/he/norinori.dm:137]: Hear me out,
```suggestion
	if(user.stat == DEAD)
		user.gib() //update it with cool effects
```
IF someone with max fortitude, KoD blessing, and Mimicry wants to pet the funny cat and get ALMOST Murdered, let them!

File: code/modules/mob/living/simple_animal/abnormality/he/norinori.dm [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+		return
+	transformed = FALSE
+	playsound(src, 'sound/effects/blobattack.ogg', 150, FALSE, 4)
+	icon_state = icon_aggro
+	attack_sound = 'sound/weapons/slashmiss.ogg'
+	SLEEP_CHECK_DEATH(10)
+	for(var/damtype in src.damage_coeff)
+		if(damtype == BRUTE)
+			continue
+		damage_coeff[damtype] += 0.4
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/he/norinori.dm:189]: If/When #1326 gets merged, remind me to change this to a dc_change

File: code/modules/paperwork/records/info/he.dm [hunk 5]
```diff
@@ -422,3 +422,13 @@
 		"Agents who witnessed the movements of Wayward Passenger reported unusually high levels of mental corruption.",
 		"When Wayward Passenger breached containment, it created spacial anomalies, dubbed T-01-170-1.",
 		"Agent Alex who entered T-01-170-1 found that it aided the suppression process.")
+
+//Norinori
+/obj/item/paper/fluff/info/he/norinori
+	abno_type = /mob/living/simple_animal/hostile/abnormality/norinori
+	abno_code = "O-02-25"
+	abno_info = list(
+		"When communicating with the abnormality as a part of Attachment Work, the employee was viciously attacked by the abnormality and killed.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/he.dm:431]:
```suggestion
		"When communicating with the abnormality as a part of Attachment Work, the employee was viciously attacked and killed by the abnormality.",
```

File: code/modules/paperwork/records/info/he.dm [hunk 6]
```diff
@@ -422,3 +422,13 @@
...
+		"When the employee initiated close physical interaction, such as petting, the Qliphoth counter was reduced by 1.",
+		"If the employee's Temperance Level was below 3, the employee was killed instead.",
+		"When the work result was Bad, the Qliphoth Counter lowered with a high probability.")
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/he.dm:434]:
```suggestion
		"When the work result was Bad, the Qliphoth Counter lowered.")
```

File: code/modules/paperwork/records/info/he.dm
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/he.dm:1]: Doesn't mention that his qliphoth lowers due to deaths at all.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/records/info/he.dm
Comment (Coxswain-Navigator) [reply code/modules/paperwork/records/info/he.dm:1]: Now that you mention it that's probably just an alpha feature that I forgot to remove later.

File: code/modules/paperwork/records/info/he.dm [hunk 7]
```diff
@@ -422,3 +422,14 @@
...
 		"When Wayward Passenger breached containment, it created spacial anomalies, dubbed T-01-170-1.",
 		"Agent Alex who entered T-01-170-1 found that it aided the suppression process.")
+
+//Norinori
+/obj/item/paper/fluff/info/he/norinori
+	abno_type = /mob/living/simple_animal/hostile/abnormality/norinori
+	abno_code = "O-02-25"
+	abno_info = list(
+		"After an employee died within the facility, the Qliphoth Counter decreased.",
+		"When communicating with the abnormality as a part of Attachment Work, the employee was viciously attacked by the abnormality and killed by the abnormality.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/he.dm:432]: That reads even worse than before.
```suggestion
		"When communicating with the abnormality as a part of Attachment Work, the employee was viciously attacked and killed by the abnormality.",
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/records/info/he.dm (hunk 7 above)
Comment (Coxswain-Navigator) [reply code/modules/paperwork/records/info/he.dm:432]: oh... whoops!

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**